packages/dataviews/src/types.ts

```diff
@@ -153,6 +154,14 @@ export type Field< Item > = {
 	 */
 	filterBy?: FilterByConfig | undefined;
 
+	/**
+	 * Validation schema for the field.
+	 */
+	validationSchema?: {
+		minLength: number;
+		maxLength: number;
+		onTouched: boolean;
```

Comment:
what is this boolean about? (it feels misnamed)

Reply:
The naming could be improved. Essentially, this boolean indicates whether the error message should be displayed only when the field is dirty or also on the first validation run.





| onTouched set to false | onTouched set to true |
|--------|--------|
|<video src="https://github.com/user-attachments/assets/1173d092-405b-4d59-8192-98ff8d311247" />  | <video src="https://github.com/user-attachments/assets/b36df728-090f-4bcf-80c6-1cbd6e4ad4b9" /> |

Reply:
I personally find it hard to see a valid use case for `onTouched` being false, so I don't think it should be controlled.

- It's unlikely for a form to mount with invalid data (invalid data shouldn't be saved to begin with), unless validation changed since persisting that value.
- An empty field should not present error messages this early, see [guideline 718 from Baymard.](https://baymard.com/premium/guideline-collections/zy6bb4/718)

I also don't think this should be on a field by field basis or within the schema (thought it does make sense for it to be there but still, it feels like a form level setting or a field type setting).